armi/reactor/components/componentParameters.py

```diff
@@ -156,9 +156,9 @@ def getHoledHexagonParameterDefinitions():
 
     pDefs = parameters.ParameterDefinitionCollection()
     with pDefs.createBuilder(location=ParamLocation.AVERAGE, saveToDB=True) as pb:
-        pb.defParam("holeOD", units="?", description="?")
+        pb.defParam("holeOD", units="cm", description="Diameter of interior hole(s)")
 
-        pb.defParam("nHoles", units="?", description="?")
+        pb.defParam("nHoles", units="cm", description="Number of interior holes")
```

Comment:
Solid addition.

Reply:
Why does the number of holes have a unit of cm?